numpy/distutils/exec_command.py

```diff
@@ -253,7 +253,13 @@ def _exec_command(command, use_shell=None, use_tee = None, **env):
     except EnvironmentError:
         # Return 127, as os.spawn*() and /bin/sh do
         return 127, ''
-    text, err = proc.communicate()
+    try:
+        text, err = proc.communicate()
```

Comment:
@dalito That's line 257:
```

    File "C:\Users\david\AppData\Local\Temp\pip-wbbc5okk-build\numpy\distutils\exec_command.py", line 256, in _exec_command
      text, err = proc.communicate()
```

Reply:
See also https://stackoverflow.com/questions/33283603/python-popen-communicate-str-encodeencoding-utf-8-errors-ignore-cr .